reference-architecture/osp-cli/ch4.4_user_data.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+#set OCP3_DOMAIN and SUBOCP3_DOMAIN to override
+OCP3_DOMAIN=${OCP3_DOMAIN:-ocp3.example.com}
+
+BASTION=bastion
+MASTERS="master-0 master-1 master-2"
+INFRA_NODES="infra-node-0 infra-node-1"
+APP_NODES="app-node-0 app-node-1 app-node-2"
+ALL_NODES="$INFRA_NODES $APP_NODES"
+ALL_HOSTS="$BASTION $MASTERS $ALL_NODES"
+
+function generate_userdata_mime() {
+  cat <<EOF
+From nobody Fri Oct  7 17:05:36 2016
+Content-Type: multipart/mixed; boundary="===============6355019966770068462=="
+MIME-Version: 1.0
+
+--===============6355019966770068462==
+MIME-Version: 1.0
+Content-Type: text/cloud-config; charset="us-ascii"
+Content-Transfer-Encoding: 7bit
+Content-Disposition: attachment; filename="bastion.yaml"
+
+#cloud-config
+hostname: $1
+fqdn: $1.$2
+
+--===============6355019966770068462==
+MIME-Version: 1.0
+Content-Type: text/x-shellscript; charset="us-ascii"
+Content-Transfer-Encoding: 7bit
+Content-Disposition: attachment; filename="allow-sudo-ssh.sh"
+
+#!/bin/sh
+sed -i "/requiretty/s/^/#/" /etc/sudoers
```

Comment:
Instead of this, why not use a drop file: https://github.com/openshift/openshift-ansible/blob/master/playbooks/aws/openshift-cluster/templates/user_data.j2#L18-L21